patches/@endo+ses-ava+0.2.27.patch

```diff
@@ -0,0 +1,250 @@
++ * @returns {T}
+  */
+-const wrapTest = (avaTest, logger = defaultLogger) => {
+-  const testerWrapper = /** @type {TesterInterface} */ (wrapTester(
+-    avaTest,
+-    logger,
+-  ));
+-  for (const methodName of testerMethodsWhitelist) {
+-    if (methodName in avaTest) {
+-      /** @type {TesterFunc} */
+-      const testerMethod = (title, implFunc, ...otherArgs) =>
+-        avaTest[methodName](title, implFunc, ...otherArgs);
+-      testerWrapper[methodName] = wrapTester(testerMethod);
+-    }
++const wrapTest = (avaTest, logError = console.error) => {
++  const sesAvaTest = augmentLogging(avaTest, logError);
++  for (const methodName of overrideList) {
++    sesAvaTest[methodName] = augmentLogging(avaTest[methodName]);
+   }
+-  harden(testerWrapper);
+-  return testerWrapper;
++  harden(sesAvaTest);
++  return sesAvaTest;
+ };
+-// harden(wrapTest);
+ export { wrapTest };
+diff --git a/node_modules/@endo/ses-ava/src/types.js b/node_modules/@endo/ses-ava/src/types.js
+deleted file mode 100644
+index 9bddf17..0000000
+--- a/node_modules/@endo/ses-ava/src/types.js
++++ /dev/null
+@@ -1,72 +0,0 @@
+-// @ts-check
+-
+-/**
+- * @typedef {(...args: unknown[]) => void} Logger
+- */
+-
+-/**
+- * @callback LogCallError
+- *
+- * Calls `thunk()` passing back approximately its outcome, but first
+- * logging any erroneous outcome to the `logger`, which defaults to
+- * `console.log`.
+- *
+- * If thunk returns a non-promise, silently return it.
+- * If thunk throws, log what was thrown and then rethrow it.
+- * If thunk returns a promise, immediately return a new unresolved promise.
+- * If the first promise fulfills, silently fulfill the returned promise even if
+- * the fulfillment was an error.
+- * If the first promise rejects, log the rejection reason and then reject the
+- * returned promise with the same reason.
+- * The delayed rejection of the returned promise is an observable difference
+- * from directly calling `thunk()` but will be equivalent enough for most
+- * purposes.
+- *
+- * TODO This function is useful independent of ava, so consider moving it
+- * somewhere and exporting it for general reuse.
+- *
+- * @param {(...unknown) => unknown} func
+- * @param {unknown[]} args
+- * @param {string} name
+- * @param {Logger=} logger
+- */
+-
+-/**
+- * Simplified form of ava's types.
+- * TODO perhaps just import ava's type declarations instead
+- * TODO reconcile also with types and API defined in avaAssertXS.js
+- *
+- * @typedef {Object} Assertions
+- * @property {(actual: unknown, message?: string) => void} assert
+- * // TODO is, deepEqual, truthy, falsy, etc...
+- */
+-
+-/**
+- * @callback BaseImplFunc
+- * This is the function that invariably starts `t => {`.
+- * Ava's types call this `Implementation`, but that's just too confusing.
+- *
+- * @param {Assertions} t
+- * @returns {unknown}
+- *
+- * @typedef {BaseImplFunc | Object} ImplFunc
+- * @property {(...unknown) => string} [title]
+- *
+- * @callback TesterFunc
```

Comment:
Yeah, this (and its dependents) are the ones I am skeptical you can omit while still using types to describe the augmentation.

Reply:
The important types here are what `wrapTest` returns and it's as simple as "whatever it got" because it shouldn't be changing the API of Ava.

Reply:
I do not understand your reply.

Reply:
I meant I don't think it's worth static typing of the helper function. But I've done it https://github.com/Agoric/agoric-sdk/pull/5774/#discussion_r923405025

Reply:
Good. Answered there.

Btw, a github weirdness that seems new to me: When I click or even hover over the link above I see that github rendered the markdown without the url fragment, thereby linking only to the PR as a whole, which confused me. So I selected "Edit" from the triple-dots of this comment to see the source, then copied the url source for this link into the url bar of a separate browser tab, and viola, I saw the comment you refer to.